environments/key4hep-common/packages.yaml

```diff
@@ -41,7 +45,7 @@ packages:
   marlin:
     require: +lccd
   madgraph5amc:
-    require: '@3: +pythia8'
+    require: '@3: +pythia8 models="loop_qcd_qed_sm_Gmu"'
```

Comment:
`madgraph5amc` doesn't seem to have a `models` variant. Is this a PR that is still open upstream?